src/components/TableLine/TableLine.jsx

```diff
@@ -1,14 +1,27 @@
 import PropTypes from 'prop-types';
+import { useLocation } from 'react-router-dom';
+import { Button } from '../Buttons/Button/Button';
+import { ButtonLink } from '../Buttons/ButtonLink/ButtonLink';
+import noop from '../../shared/noop';
 
-export const TableLine = ({ number, name, children, ...restProps }) => {
+export const TableLine = ({ number, name, children, btnHandleClick, path, isButtonLink, ...restProps }) => {
```

Comment:
just `onClick` or `handleClick` is not enough for you ? don't create difficult names where you do not need it

Reply:
Thx, for the comment. I changed it.
```suggestion
export const TableLine = ({ number, name, children, handleClick, path, isButtonLink, ...restProps }) => {
```